content/en/docs/tasks/security/authz-tcp/index.md

```diff
@@ -105,85 +134,42 @@ Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpa
 * The **Book Reviews** section on the lower right of the page includes an error message **"Ratings service is
   currently unavailable"**.
 
-This is because Istio authorization is "deny by default", which means that you need to explicitly
-define access control policies to grant access to the MongoDB service.
+## Enforce workload-level access control for TCP traffic
```

Comment:
Please see my comment and suggestions regarding the previous heading. The suggestion below provides the context for the command just indented properly to ensure the numbered list remains unbroken and before the command to give readers the needed context to decide whether or not they want to run the command. Please follow this pattern in this and all other tasks.
 ```suggestion
    After configuring all requests be denied by default, we need to create a `bookinfo-ratings-v2` policy that lets
    requests coming from the `cluster.local/ns/default/sa/bookinfo-ratings-v2` service account through to the MongoDB
    workload at port `27017`. We grant access to the service account, because requests coming from the `ratings-v2`
    workload are issued using the `cluster.local/ns/default/sa/bookinfo-ratings-v2` service account. 
```

Reply:
Are you suggesting to remove the heading `## Enforce workload-level access control for TCP traffic`? This makes the page looks really weird, the only headings in the page are `Before you begin`. `Installing and configuring a TCP workload` and `Cleanup`.

Also, I don't think the indentation is correct, that will make the text belong to the previous numbered item on the page.

Reply:
Yes, that is the idea. You want the entire procedure to be a single flow. Why do you want to have it broken up into single step sections? Interrupting the flow of the procedure with a heading takes readers from a "Do" mindset to a "Understand" mindset. You want to avoid forcing readers to context switch unless it is significant. Enforcing the workload-level access control is part of the overarching task: Installing and configuring authentication for a TCP workload. The text indentation is needed to ensure that the numbers continue uninterrupted throughout.

Reply:
I don't think this is a good idea. That means we also have to remove other headings like `Before you begin`, `Installing and configuring a TCP workload` and `Cleanup`. Each section has its own responsibility and these are all about "Do", if the user want to learn the concept, they go to the concept page instead.

`Installing and configuring authentication for a TCP workload` is one step in the task to prepare for the environment, `Enforcing the workload-level access control` is not part of it.

The text indentation could bring more confusion because it groups the contents with the **previous** numbered item. If you look at the page, it's very strange as it seems we are putting non related descriptions in the numbered item because these should go to under the numbered item instead of before it.